### ISSUE TITLE
Support for multiple email addresses.

### ISSUE THREAD

**Issue by Zimmi48** [tournesol-app/tournesol#1151]
I just created a Tournesol account and when I went to my user settings I learned that my email address (a Gmail address) was not considered trusted. Since I belong to an academic institution and have an email address from a trusted domain, I used the "New email address" field to add this address to my account. But doing so, I discovered that my email address had actually been changed!

So this issue is twofold:

1. From a usability point of view, the choice of the words "New email address" do not indicate sufficiently clearly that this will replace the email address in use (and thus with which you can identify) instead of adding to the list of known email addresses.
2. It would be better to support saving multiple email addresses for a single account. For instance, in my case, I prefer to use my personal Gmail address for anything that is not work-related, but I am fine with adding a professional email address as a secondary email address, e.g., in order to verify my account.

**Comment by GresilleSiffle** [comment]
Hello @Zimmi48 

Thanks for sharing this issue.

> From a usability point of view, the choice of the words "New email address" do not indicate sufficiently clearly that this will replace the email address in use (and thus with which you can identify) instead of adding to the list of known email addresses.

It's true that `New email address` can be misleading. At the same time, this field's placeholder is displayed in a section called `Change email address` which reflects what will happen in you type a new address and click on the submit button.

What about renaming the section `Change email address` to `Change my email address`? This accentuates that the form is working with only one email address, meaning any typed addresses will replace the previous one.

We can also think about changing the placeholder text, from `New email address` to something else. I think simply using `Email address` could be misleading too, as it doesn't indicate which kind of address the user should type. Its current one? A new one?

A third option could be to add a small sentence before the input:  `Use the input below to replace your current email address by a new one`. This option has currently my preference.

**Comment by Zimmi48** [comment]
> It's true that `New email address` can be misleading. At the same time, this field's placeholder is displayed in a section called `Change email address` which reflects what will happen in you type a new address and click on the submit button.
> 
> What about renaming the section `Change email address` to `Change my email address`? This accentuates that the form is working with only one email address, meaning any typed address will replace the previous one.

Yes, probably it would help a little. But my experience also shows that I probably missed the section title entirely.

I guess the problem is not in fact with the "New email address" field but with the "Send verification email" field. Clicking on this button does not highlight the fact that the previous email address will be removed. What about replacing the button text with "Update email" (consistent with "Update password" below) or "Change email address"? That you get a confirmation email is not surprising. It's probably not needed to highlight it in the button text...

>Also, feel free to use your non-trusted email address on Tournesol. We recently deployed a vouching system to allow users with non-trusted email addresses to gain more voting right. If you have friends using Tournesol, do not hesitate to vouch for each other.

I've noticed thanks, but I don't yet have friends that I know are already using Tournesol.
